tests/scripts/test_format_lintro_pr_comment.py

```
@@ -0,0 +1,143 @@
+    module = _load_module()
+    job_log = tmp_path / "lint-job.log"
+    job_log.write_text(
+        "\n".join(
+            [
+                "🛠️ Lintro Code Quality\tUNKNOWN STEP\t2026-04-17T09:03:31.145Z\tprefix",
+                "🛠️ Lintro Code Quality\tUNKNOWN STEP\t2026-04-17T09:03:31.146Z\t📋 EXECUTION SUMMARY",
+                "🛠️ Lintro Code Quality\tUNKNOWN STEP\t2026-04-17T09:03:31.147Z\t| Tool | Status |",
+                "🛠️ Lintro Code Quality\tUNKNOWN STEP\t2026-04-17T09:03:31.148Z\t| black | PASS |",
+                "🛠️ Lintro Code Quality\tUNKNOWN STEP\t2026-04-17T09:03:31.149Z\tCleaning up orphan processes",
+            ],
+        ),
```

Ruff (0.15.10)

[warning] Consider f-string instead of string join

Replace with f-string

(FLY002)